### ISSUE TITLE
Error on tab complete within function call

### ISSUE THREAD

**Issue by meriops** [randy3k/radian#154]
Recently, I get an error message `"Error in getAnywhere(x) : reached CPU time limits"` when I try to auto-complete the name of a data frame from within a function call. For example, if I start `radian` and try to see the head of the `mtcars` data frame by typing `head(|)`, followed by `mt`, as in `head(mt)`, then I get the error. It happens only with `radian`, not with the R console. And I don't really have any clue where it comes from, to be honest... Any help would be appreciated!

**Comment by randy3k** [comment]
What version of radian is it? Try updating it first. There was some known related issues in old versions of radian.

**Comment by meriops** [comment]
It's actually v0.4.6-1 already...

**Comment by randy3k** [comment]
By the way， what os?

**Comment by meriops** [comment]
Yes, sorry for not giving that info earlier. It is archlinux...

**Comment by randy3k** [comment]
I still cannot reproduce it with a docker archlinux container. Did you specify any options in Rprofile related to radian?

**Comment by randy3k** [comment]
Not sure if it is related to dafbadb

Would you try the current master
```
pip install -U git+https://github.com/randy3k/radian
```

**Comment by meriops** [comment]
The error still occurs with the current master, but yes, I have radian options specified in  `.Rprofile`. I will try to activate / deactivate these options and see if it changes anything...

**Comment by meriops** [comment]
It _seems_ that the error was caused by setting `options(radian.completion_timeout = 0.05)` in the `.Rprofile`, as on the README. Any value above 0.5 seems to work fine though.

**Comment by randy3k** [comment]
But it should not give you an error when cancelling the timeout.

**Comment by meriops** [comment]
Indeed, cancelling with value equal to 0 does not give an error. It seems that only values between 0 and 0.05 return the error.

**Comment by randy3k** [comment]
I still cannot reproduce it when smaller values of timeout on archlinux. Could it be the version of R? I have R 3.6.2.

**Comment by meriops** [comment]
It's not the R version, as I also have 3.6.2. But I think I found the culprit... I load [jalvesaq/colorout](https://github.com/jalvesaq/colorout) and set its `setOutputColor()` in my `.Rprofile` and it seems that it causes the error with `radian`. I don't really know _why_ though...

**Comment by randy3k** [comment]
Thanks, let me see if I could reproduce it

**Comment by randy3k** [comment]
Yeah, I could reproduce it.

**Comment by randy3k** [comment]
In radian, the output channel is suppressed to avoid error messages from the timeout. However, `colorout` has masked the default output channel and print directly to stdout/stderr.

We could use `tryCatch` to catch the message a bit earlier.

**Comment by randy3k** [comment]
https://github.com/randy3k/rchitect/blob/640170ab7bec40837e1e6399782c894b623ca7ca/rchitect/completion.py#L27-L27 should fix it

To install the dev version of rchitect
```
pip install -U git+https://github.com/randy3k/rchitect
```

**Comment by meriops** [comment]
Indeed, it seems to work with the dev version of rchitect. Thanks for that!
